modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/utilities/index.js

```diff
@@ -68,6 +85,7 @@ export function getValueFromItem(item, fieldName) {
 
 export function executeAsyncAction(url, method = 'GET') {
 	return fetch(url, {
+		headers: fetchHeaders,
```

Comment:
Can we inline `fetchHeaders` in here? It's only used once, and the overhead of creating it multiple times is insignificant IMO. Plus, if we ever get to the point where we can change languages using SPA (without reloading the page) than the language will always be up to date.